tests/scripts/pre-validation.sh

```diff
@@ -10,6 +10,7 @@ DA_DIR="solutions/banking"
 TERRAFORM_SOURCE_DIR="tests/resources/existing-resources"
 JSON_FILE="${DA_DIR}/catalogValidationValues.json"
 REGION="us-south"
+SECRETS_MANAGER_GUID="79c6d411-c18f-4670-b009-b0044a238667"
```

Comment:
can you pull this from the permanent resource yaml instead of hard coding it incase it ever changes